Buttplug.Server/Util/FleshlightHelper.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Buttplug.Server.Util
+{
+    public class FleshlightHelper
+    {
+        /// <summary>
```

Comment:
YAY A COMMENT.

Reply:
I will make you regret asking for comments in a future PR. Mah ha ha ha